rules/network/command_and_control_accepted_default_telnet_port_connection.toml

```
@@ -45,10 +44,11 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset: network_traffic.flow and event.type: connection
-  and not event.action:(
-      flow_dropped or denied or deny or
-      flow_terminated or timeout or Reject or network_flow)
+(event.dataset: network_traffic.flow or event.category: (network or network_traffic))
+    and event.type: connection and not event.action:(
+        flow_dropped or denied or deny or
+        flow_terminated or timeout or Reject or network_flow)
+    and destination.port:23
```

nit: for kql, use no spaces between field:value

```suggestion
(event.dataset:network_traffic.flow or event.category:(network or network_traffic))
    and event.type:connection and not event.action:(
        flow_dropped or denied or deny or
        flow_terminated or timeout or Reject or network_flow)
    and destination.port:23
```

Also, if we are going to refine this to `or` for event category, we should just drop the dataset check, since it is _also_ a `network` event

---

> Also, if we are going to refine this to or for event category, we should just drop the dataset check, since it is also a network event anyway

Reminder that we rely on `event.dataset` to point to the NPC integration for `related_integrations` and the `flow` data stream. This is also important for field validation within the integration.